src/python/grpcio/grpc/_cython/cygrpc.pxd

```diff
@@ -37,11 +37,10 @@ include "_cygrpc/grpc_gevent.pxd.pxi"
 IF UNAME_SYSNAME != "Windows":
     include "_cygrpc/fork_posix.pxd.pxi"
 
-IF ASYNCIO_SUPPORT:
-    include "_cygrpc/aio/iomgr/socket.pxd.pxi"
-    include "_cygrpc/aio/iomgr/timer.pxd.pxi"
-    include "_cygrpc/aio/iomgr/resolver.pxd.pxi"
-    include "_cygrpc/aio/grpc_aio.pxd.pxi"
-    include "_cygrpc/aio/callbackcontext.pxd.pxi"
-    include "_cygrpc/aio/call.pxd.pxi"
-    include "_cygrpc/aio/channel.pxd.pxi"
+include "_cygrpc/aio/iomgr/socket.pxd.pxi"
```

Comment:
nit: A short comment will help.

Reply:
what do you want to add?

Reply:
Something like: Following `.pxi` files are used for async API.
So they are logically separated from above `pxi` files.